test/performance/odp_sched_perf.c

```diff
@@ -507,6 +509,13 @@ static void print_options(test_options_t *options)
 	printf("  queue type                %s\n\n", options->queue_type == 0 ? "parallel" :
 						     options->queue_type == 1 ? "atomic" :
 						     "ordered");
+
+	printf("Extra rd/rw ops per event (queue ctx + uarea + data)\n");
```

Comment:
Perhaps the same terms that are used elsewhere in output could be used here: "queue ctx" -> "queue context", "uarea" -> "user area", "data" -> "event data".

Reply:
Changed on V2.